.gitignore

```diff
@@ -95,6 +95,9 @@ qtcreator-*
 
 *~
 
+#clion
+.idea
```

Comment:
Please don't remove this

Reply:
What do you mean? Are these files somehow connected to the code?

Reply:
This excludes the `.idea/` folder created by CLion from ever getting committed accidentally.

Reply:
I got confused. Nevermind.